AutomatedTesting/Gem/PythonTests/Physics/TestSuite_InDevelopment.py

```diff
@@ -26,154 +26,133 @@ class TestAutomation(TestAutomationBase):
         from .tests.material import Material_DefaultMaterialLibraryChangesWork as test_module
-        self._run_test(request, workspace, editor, test_module, enable_prefab_system=False)
+        self._run_test(request, workspace, editor, test_module)
 
     def test_Collider_SameCollisionGroupSameLayerCollide(self, request, workspace, editor, launcher_platform):
         from .tests.collider import Collider_SameCollisionGroupSameLayerCollide as test_module
-        self._run_test(request, workspace, editor, test_module, enable_prefab_system=False)
+        self._run_test(request, workspace, editor, test_module)
 
     def test_Ragdoll_OldRagdollSerializationNoErrors(self, request, workspace, editor, launcher_platform):
         from .tests.ragdoll import Ragdoll_OldRagdollSerializationNoErrors as test_module
-        self._run_test(request, workspace, editor, test_module, enable_prefab_system=False)
+        self._run_test(request, workspace, editor, test_module)
 
     @fm.file_override("default.physxconfiguration", "ScriptCanvas_OverlapNode.physxconfiguration")
     def test_ScriptCanvas_OverlapNode(self, request, workspace, editor, launcher_platform):
         from .tests.script_canvas import ScriptCanvas_OverlapNode as test_module
-        self._run_test(request, workspace, editor, test_module, enable_prefab_system=False)
+        self._run_test(request, workspace, editor, test_module)
 
     def test_Material_StaticFriction(self, request, workspace, editor, launcher_platform):
         from .tests.material import Material_StaticFriction as test_module
-        self._run_test(request, workspace, editor, test_module, enable_prefab_system=False)
+        self._run_test(request, workspace, editor, test_module)
         
     @fm.file_revert("c4888315_material_addmodifydeleteoncollider.physmaterial",
                     r"AutomatedTesting\Levels\Physics\Material_LibraryCrudOperationsReflectOnCollider")
     def test_Material_LibraryCrudOperationsReflectOnCollider(self, request, workspace, editor, launcher_platform):
         from .tests.material import Material_LibraryCrudOperationsReflectOnCollider as test_module
-        self._run_test(request, workspace, editor, test_module, enable_prefab_system=False)
+        self._run_test(request, workspace, editor, test_module)
 
     @fm.file_revert("c15563573_material_addmodifydeleteoncharactercontroller.physmaterial",
                     r"AutomatedTesting\Levels\Physics\Material_LibraryCrudOperationsReflectOnCharacterController")
     def test_Material_LibraryCrudOperationsReflectOnCharacterController(self, request, workspace, editor, launcher_platform):
         from .tests.material import Material_LibraryCrudOperationsReflectOnCharacterController as test_module
-        self._run_test(request, workspace, editor, test_module, enable_prefab_system=False)
-
-    @fm.file_revert("c4888315_material_addmodifydeleteoncollider.physmaterial",
```

Comment:
What happened to these two tests? I don't think I saw them mentioned in the description.

Reply:
There are some duplicated code in InDevelopment.

Reply:
Those two tests have exactly same copy in the test suite.